exchange/bidder.go

```diff
@@ -176,13 +176,16 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, bidderRequest Bidde
 				reqData[i].Headers.Add("Sec-GPC", reqInfo.GlobalPrivacyControlHeader)
 			}
 			if bidRequestOptions.addCallSignHeader {
+				startSignRequestTime := time.Now()
 				signatureMessage, err := adsCertSigner.Sign(reqData[i].Uri, reqData[i].Body)
+				bidder.me.RecordAdsCertSignTime(time.Since(startSignRequestTime))
```

Comment:
Do we expect that the time taken to generate a signature to be significant to warrant tracking as a metric?

Reply:
When we enable this feature it can help to show if this feature can be a potential bottleneck in request handling. It can be especially valuable when PBS uses remote signatory service.  
@SyntaxNode please confirm